Library/Homebrew/rubocops/components_order.rb

```diff
@@ -3,6 +3,7 @@
 
 require "ast_constants"
 require "rubocops/extend/formula_cop"
+require "utils/array"
```

Comment:
Note I put the util in a separate file, despite being at the top `Utils` level, so that this rubocop could import just the method it needs, and not the bulk of the `Utils` namespace. As a bonus, rubocop no longer has a direct `active_support` require, but still makes use of active_support methods such as `present?`

Reply:
@dduugg IMO it's worth either just importing all of `Utils` in `rubocop` or moving this to another namespace, I think defining modules over multiple files is a confusing behaviour for non-Rubyists (as you don't see the same in most other ecosystems).